packages/ts/generator-typescript-plugin-backbone/src/EndpointMethodRequestBodyProcessor.ts

```diff
@@ -18,54 +18,87 @@ export type EndpointMethodRequestBody = ReadonlyDeep<OpenAPIV3.RequestBodyObject
 export type EndpointMethodRequestBodyProcessingResult = Readonly<{
   parameters: readonly ParameterDeclaration[];
   packedParameters?: ObjectLiteralExpression;
+  initParam: ts.Identifier;
 }>;
 
 export default class EndpointMethodRequestBodyProcessor {
+  readonly #defaultInitParamName = 'init';
+
   readonly #dependencies: DependencyManager;
   readonly #owner: Plugin;
   readonly #requestBody?: EndpointMethodRequestBody;
+  readonly #endpointRequestInitIdentifier: ts.Identifier;
 
   public constructor(
     requestBody: ReadonlyDeep<OpenAPIV3.ReferenceObject | OpenAPIV3.RequestBodyObject> | undefined,
     dependencies: DependencyManager,
     owner: Plugin,
+    endpointRequestInitIdentifier: ts.Identifier,
   ) {
     this.#owner = owner;
     this.#dependencies = dependencies;
     this.#requestBody = requestBody ? owner.resolver.resolve(requestBody) : undefined;
+    this.#endpointRequestInitIdentifier = endpointRequestInitIdentifier;
   }
 
   public process(): EndpointMethodRequestBodyProcessingResult {
     if (!this.#requestBody) {
       return {
-        parameters: [],
+        parameters: [
+          ts.factory.createParameterDeclaration(
+            undefined,
+            undefined,
+            undefined,
+            this.#defaultInitParamName,
+            ts.factory.createToken(ts.SyntaxKind.QuestionToken),
+            ts.factory.createTypeReferenceNode(this.#endpointRequestInitIdentifier),
+          ),
+        ],
+        packedParameters: ts.factory.createObjectLiteralExpression(),
+        initParam: ts.factory.createIdentifier(this.#defaultInitParamName),
       };
     }
 
     const parameterData = this.#extractParameterData(this.#requestBody.content[defaultMediaType]?.schema);
+    let initParamName = this.#defaultInitParamName;
+
+    while (parameterData.map(([name]) => name).includes(initParamName)) {
```

Comment:
You should use `find` function here:
```suggestion
    while (parameterData.find(([name]) => name === initParamName))) {
```

**P.S.** Though, if you go with `createUniqueIdentifier`, this part will disappear.